clang/lib/Driver/Driver.cpp

```diff
@@ -5945,6 +5952,9 @@ InputInfo Driver::BuildJobsForActionNoCache(
                                              AtTopLevel, MultipleArchs,
                                              OffloadingPrefix),
                        BaseInput);
+    if (JobForPreprocessToStdout) {
+      ResultForPreprocessToStdout = InputInfo(A, "-", BaseInput);
+    }
```

Comment:
```suggestion
    if (JobForPreprocessToStdout)
      ResultForPreprocessToStdout = InputInfo(A, "-", BaseInput);
```

Reply:
Modified the patch  and all pre-check-in pass already.